transport/tchannel/example_test.go

```diff
@@ -0,0 +1,44 @@
+package tchannel_test
+
+import (
+	"log"
+
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/transport/tchannel"
+)
+
+func ExampleInbound() {
+	transport, err := tchannel.NewTransport(tchannel.ServiceName("myservice"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	dispatcher := yarpc.NewDispatcher(yarpc.Config{
+		Name:     "myservice",
+		Inbounds: yarpc.Inbounds{transport.NewInbound()},
```

Comment:
This is not a change. The port is attributed to the Transport and defaults to :0 if there’s no ListenAddr.